models/src/main/kotlin/io/provenance/api/models/p8e/contracts/SmartContractQueryRequest.kt

```diff
@@ -0,0 +1,11 @@
+package io.provenance.api.models.p8e.contracts
+
+import io.provenance.api.models.account.AccountInfo
+import io.provenance.api.models.p8e.ProvenanceConfig
+
+data class SmartContractQueryRequest(
```

Comment:
I created two because I see the query and transaction requests diverging as we add more functionality.  I envision the query eventually having a parameter like throwing exception if not found.  And the transactions request taking on a few transaction specific options.